dashboard/migrations/0017_auto_20200720_1203.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.14 on 2020-07-20 16:03
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dashboard', '0015_auto_20200116_1408'),
+    ]
+
+    operations = [
+        # We have to remove all rows from resource and resource access before performing this migration
+        # As there is no way of knowing which course_id a resource access was linked to
+        # This is the current process for the cron anyway
+        migrations.RunSQL("DELETE FROM resource_access;"),
+        migrations.RunSQL("DELETE FROM resource;"),
+        migrations.RemoveField(
+            model_name='resource',
+            name='course_id',
+        ),
+        migrations.AddField(
+            model_name='resourceaccess',
```

Comment:
The model name is spelled wrong here must be `resource_access'?

Reply:
Probably this is django generated. I see migrations applied correctly

Reply:
Yeah it was generated by Django. It looks like it's the name of the class, case insensitive.